gallery/streamlit_patches.py

```diff
@@ -314,9 +308,6 @@ def _get_code_from_path(script_path: str) -> Any:
     with source_util.open_python_file(script_path) as f:
         filebody = f.read()
 
-    if config.get_option("runner.magicEnabled"):
```

Comment:
Is this no longer necessary to add magic? Or does it not work for some reason?

Reply:
This logic moved to another file in Streamlit, and I'm not sure if it is working in the current gallery app. But I haven't seen it used anywhere in the gallery app, so might not be super urgent to fix it. But I will do another check into this

Reply:
I added it back to the patch 👍